### ISSUE TITLE
tarantoolctl silently fails when starting an instance with '-' in its name under centos7

### ISSUE THREAD

**Issue by vstarodub** [tarantool/tarantool#2552]
tarantoolctl silently fails when starting an instance with '-' in its name under centos7. Console output looks fine. Logs contain only "Started Tarantool Database Server" without any further diagnostic messages.

How to reproduce using example.lua from tarantool rpm:

```
# tarantoolctl start example
Starting instance...
Forwarding to 'systemctl start tarantool@example'
# tarantoolctl enter example
connected to unix/:/var/run/tarantool/example.control
unix/:/var/run/tarantool/example.control>
# tarantoolctl stop example

# cp /etc/tarantool/instances.available/example{,-1}.lua
# tarantoolctl start example-1
Starting instance...
Forwarding to 'systemctl start tarantool@example-1'

# journalctl -r | head
-- Logs begin at Sun 2017-05-14 05:00:41 MSK, end at Wed 2017-06-28 14:34:05 MSK. --
Jun 28 14:34:05 bla.bla systemd[1]: Started Tarantool Database Server.
<...>

# tarantoolctl enter example-1
Can't connect to /var/run/tarantool/example-1.control (No such file or directory)
```

**Comment by rtsisyk** [comment]
This is a systemd limitation, let's update documentation for this case.
https://github.com/tarantool/doc/issues/276

**Comment by kostja** [comment]
It shouldn't *silently* fail. We need to check instance name and give a warning.

**Comment by rtsisyk** [comment]
Fixed by #2684

**Comment by kostja** [comment]
If you fixed it, please remove from the docs that this is not supported.
